lib/gollum/assets.rb

```diff
@@ -19,6 +19,19 @@ def self.sprockets(dir = File.dirname(File.expand_path(__FILE__)))
       env.js_compressor  = ::Precious::Assets::JS_COMPRESSOR if defined?(::Precious::Assets::JS_COMPRESSOR)
       env.css_compressor = :sassc
 
+      options = {
+        sass_config: {
+          quiet_deps: true,
+          silence_deprecations: [
```

Comment:
What are these deprecation messages saying? Shouldn't we address them rather then ignoring the messages?